### PR TITLE
[SYCLomatic][DOC] Update DPCT1028 "Detailed Help" and "Suggestions to Fix" in doc

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by zhimingwang36 — APPROVED** [review]
LGTM

File: docs/dev_guide/diagnostic_ref/dpct1028.rst [hunk 1]
```diff
@@ -16,10 +16,14 @@ Detailed Help
...
+as an engine type or a CUDA\* device attribute.
 
 Suggestions to Fix
 ------------------
 
+If the parameter is an engine type:
 Rewrite this code manually by using a supported random number generator. See the
 `Random Number Generators <https://software.intel.com/content/www/us/en/develop/documentation/oneapi-mkl-dpcpp-developer-reference/top/random-number-generators/manual-offload-rng-routines/engines-basic-random-number-generators.html>`_ topic for more information.
+
+If the parameter is a CUDA\* device attribute:
```
Comment (krisdale) [inline docs/dev_guide/diagnostic_ref/dpct1028.rst]: remove "\*" (only need to use attribution on first use in page).

File: docs/dev_guide/diagnostic_ref/dpct1028.rst [hunk 2]
```diff
@@ -16,10 +16,14 @@ Detailed Help
...
+Redesign the code logic to get rid of the usage of the device attribute.
```
Comment (krisdale) [inline docs/dev_guide/diagnostic_ref/dpct1028.rst]: "Redesign the code logic to remove use of the device attribute."

**Review by krisdale — CHANGES_REQUESTED** [review]
Approved if the requested changes are implemented.

**Review by zhimingwang36 — APPROVED** [review]

**Pull request merged**